lib/node_modules/@stdlib/math/base/special/gcdf/docs/repl.txt

```diff
@@ -0,0 +1,30 @@
+
+{{alias}}( a, b )
+    Computes the greatest common divisor (gcd).
```

Comment:
This description must be updated as well, to reflect that this package is for single-precision floating point numbers.